docs/source/develop/api/queries.rst

```diff
@@ -15,6 +15,89 @@ The validation for all queries includes:
 - query counter — checked to be incremented with every subsequent query from query creator
 - roles — depending on the query creator's role: the range of state available to query can relate to to the same account, account in the domain, to the whole chain, or not allowed at all
 
+Engine Receipts
+^^^^^^^^^^^^^^^
+
+Purpose
+-------
+
+Retrieve a receipt of a CallEngine command.
+Similar to the eth.GetTransactionReceipt API call of Ethereum JSON RPC API.
+Allows to access the event log created during computations inside the EVM.
+
+Request Schema
+--------------
+
+.. code-block:: proto
+
+   message GetEngineReceipts{
+    string tx_hash = 1;     // hex string
+}
+
+Request Structure
+-----------------
+
+.. csv-table::
+    :header: "Field", "Description", "Constraint", "Example"
+    :widths: 15, 30, 20, 15
+
+    "Transaction Hash", "hash of the transaction that has the CallEngine command", "hash in hex format", "5241f70cf3adbc180199c1d2d02db82334137aede5f5ed35d649bbbc75ab2634"
+
+Response Schema
+---------------
+
+.. code-block:: proto
+
+    message EngineReceiptsResponse {
+        repeated EngineReceipt engine_receipt = 1;
+    }
+    message EngineReceipt {
```

Comment:
This interface doesn't match the proto anymore after changes in the latter. Needs to be aligned with the source.

Reply:
@ekovalev  can you please check now?